### PR TITLE
Remove references to Windows 7

### PR DESCRIPTION
**Pull request by Daltz333**
Windows 7 and 32bit OSes are no longer supported.

### CODE REVIEW

File: source/docs/zero-to-robot/step-2/wpilib-setup.rst [hunk 1]
```diff
@@ -29,13 +27,6 @@ When you download the WPILib installer, it is distributed as a disk image file `
 
       .. note:: Other installed programs may associate with iso files and the :guilabel:`mount` option may not appear. If that software does not give the option to mount or extract the iso file, then follow the directions in the "Windows 7" tab.
```
Comment (sciencewhiz) [inline source/docs/zero-to-robot/step-2/wpilib-setup.rst:28]: Can't delete the Windows 7 tab because of this

**Comment by TheTripleV** [comment]
Needs a rebase? Seems to undo prior PRs.

**Comment by Daltz333** [comment]
Fixed.

File: source/docs/zero-to-robot/step-2/wpilib-setup.rst [hunk 2]
```diff
@@ -22,16 +20,15 @@ When you download the WPILib installer, it is distributed as a disk image file `
...
+
       Windows 10+ users can right click on the downloaded disk image and select :guilabel:`Mount` to open it. Then launch ``WPILibInstaller.exe``.
 
       .. image:: images/wpilib-setup/extract-windows-10.png
          :alt: The menu after right clicking on an .iso file to choose "Mount".
 
-      .. note:: Other installed programs may associate with iso files and the :guilabel:`mount` option may not appear. If that software does not give the option to mount or extract the iso file, then follow the directions in the "Windows 7" tab.
-
-   .. group-tab:: Windows 7
+      .. note:: Other installed programs may associate with iso files and the :guilabel:`mount` option may not appear. If that software does not give the option to mount or extract the iso file, then follow the directions below.
```
[the remark this answers is not in the document]
Comment (sciencewhiz) [reply source/docs/zero-to-robot/step-2/wpilib-setup.rst:29]: Actually, I see that other places not changed by this PR

[the remark this answers is not in the document]
Comment (Daltz333) [reply source/docs/zero-to-robot/step-2/wpilib-setup.rst:29]: Yes. This is an issue with tabs and images.

**Review by TheTripleV — APPROVED** [review]

**Pull request merged**